packages/components/psammead-most-read/src/Item/index.jsx

```diff
@@ -85,10 +85,11 @@ export const MostReadLink = ({
   script,
   title,
   href,
+  small,
   children,
 }) => (
   <StyledItem dir={dir}>
-    <StyledLink href={href} script={script} service={service}>
+    <StyledLink href={href} script={script} service={service} small={small}>
```

Comment:
Not very keen on the word `small`... it might seem quite vague to someone looking at this for the first time. Would be open to suggestions.